packages/teams-js/test/public/pages.spec.ts

```diff
@@ -210,42 +335,917 @@ describe('AppSDK-TeamsAPIs', () => {
+          expect(promise).resolves.toBeUndefined();
+        });
+      });
+    });
+
+    describe('Testing pages.tabs.getMruTabInstances function', () => {
+      let expectedTabInstanceParameters: TabInstanceParameters = {
+        favoriteChannelsOnly: true,
+        favoriteTeamsOnly: true,
+      };
+
+      it('pages.tabs.getMruTabInstances should not allow calls before initialization', async () => {
+        await expect(() => pages.tabs.getMruTabInstances()).rejects.toThrowError(
+          'The library has not yet been initialized',
+        );
+      });
+
+      Object.values(FrameContexts).forEach(context => {
+        it(`pages.tabs.getMruTabInstances should successfully getTabInstance when no parameters are passed and initialized with ${context} context`, async () => {
+          await utils.initializeWithContext(context);
+          const promise = pages.tabs.getMruTabInstances();
+          let message = utils.findMessageByFunc('getMruTabInstances');
+
+          utils.respondToMessage(message, expectedTabInstanceParameters);
+          expect(message).not.toBeNull();
+          expect(promise).resolves.toBe(expectedTabInstanceParameters);
+        });
+
+        it(`pages.tabs.getMruTabInstances should be undefined getTabInstance when parameters are passed and  initialized with ${context} context`, async () => {
+          await utils.initializeWithContext(context);
+          const promise = pages.tabs.getMruTabInstances(expectedTabInstanceParameters);
+          let message = utils.findMessageByFunc('getMruTabInstances');
+
+          utils.respondToMessage(message, expectedTabInstanceParameters);
+          expect(message).not.toBeNull();
+          expect(promise).resolves.toBe(expectedTabInstanceParameters);
+        });
+
+        it(`pages.tabs.getMruTabInstances should be undefined when no parameters are passed and initialized with ${context} context`, async () => {
+          await utils.initializeWithContext(context);
+          const promise = pages.tabs.getMruTabInstances();
+          let message = utils.findMessageByFunc('getMruTabInstances');
+
+          utils.respondToMessage(message);
+          expect(message).not.toBeNull();
+          expect(promise).resolves.toBeUndefined();
+        });
+      });
+    });
+
+    describe('Testing pages.tabs.isSupported function', () => {
+      it('pages.tabs.isSupported should return false if the runtime says pages.tabs is not supported', () => {
+        utils.setRuntimeConfig({ apiVersion: 1, supports: { pages: {} } });
+        expect(pages.tabs.isSupported()).not.toBeTruthy();
+      });
+
+      it('pages.tabs.isSupported should return true if the runtime says pages.tabs is supported', () => {
+        utils.setRuntimeConfig({
+          apiVersion: 1,
+          supports: { pages: { tabs: {} } },
+        });
+        expect(pages.tabs.isSupported()).toBeTruthy();
+      });
+    });
+  });
+
+  describe('Testing pages.config namespace', () => {
```

Comment:
I just noticed that we have a test file called config.spec.ts, which apperas to have pages.config/settings capability tests in it. Could you please take a look at that file and reconcile it with these changes?

Reply:
I noticed that both are almost the same and we can keep both of them and it's not going to be a problem. But to avoid redundancy I can remove the `config.spec.ts`.

Reply:
I've removed the duplicate files in the code base and updated in the latest commit